### ISSUE TITLE
Cannot create Field element because property reactFinalForm is missing

### ISSUE THREAD

**Issue by morloy** [final-form/react-final-form#374]
### Are you submitting a **bug report** or a **feature request**?

Bug report

### What is the current behavior?

Flow gives an error when creating a form field:
`Cannot create Field element because property reactFinalForm is missing in props [1] but exists in object type [2].`

### What is the expected behavior?

No error.

### What's your environment?

```
    "final-form": "4.11.0",
    "react-final-form": "4.0.0",
```

**Comment by erikras** [comment]
Published fix in [`v4.0.1`](https://github.com/final-form/react-final-form/releases/tag/v4.0.1).

**Comment by morloy** [comment]
@erikras just noticed that the same happens with `FormSpy`.

**Comment by erikras** [comment]
Duh. Sorry about that. Published fix in [`v4.0.2`](https://github.com/final-form/react-final-form/releases/tag/v4.0.2).
